mni_atlas_reader/atlas_reader.py

```diff
@@ -419,32 +428,51 @@ def create_output(filename, atlas, voxelThresh=2, clusterExtend=5,
     cluster_mean = []
     volume_summary = []
 
-    for c in coords:
+    # get template image for plotting cluster maps
+    bgimg = resource_filename('mni_atlas_reader',
+                              'data/templates/MNI152_T1_1mm_brain.nii.gz')
+
+    for n, coord in enumerate(coords):
+        print(coord)
         peakinfo = get_peak_info(
-            c, atlastype=atlas, probThresh=probabilityThreshold)
+            coord, atlastype=atlas, probThresh=probabilityThreshold)
         peak_summary.append([p[1] if type(p[1]) != list else '; '.join(
             ['% '.join(e) for e in np.array(p[1])]) for p in peakinfo])
-        voxID = get_vox_coord(img.affine, c)
+        voxID = get_vox_coord(img.affine, coord)
         peak_value.append(imgdata[voxID[0], voxID[1], voxID[2]])
 
-        idx = get_vox_coord(img.affine, c)
+        idx = get_vox_coord(img.affine, coord)
         clusterID = clusters[idx[0], idx[1], idx[2]]
         clusterinfo = get_cluster_info(
             clusters == clusterID,
             img.affine,
             atlastype=atlas,
             probThresh=probabilityThreshold)
         cluster_summary.append(['; '.join(
-            ['% '.join([str(round(e[0], 2)), e[1]]) for e in c[1]]) for c in
-             clusterinfo])
+            ['% '.join([str(round(e[0], 2)), e[1]]) for e in coord[1]]) for
+             coord in clusterinfo])
         cluster_mean.append(imgdata[clusters == clusterID].mean())
 
         voxel_volume = int(img.header['pixdim'][1:4].prod())
         volume_summary.append(np.sum(clusters == clusterID) * voxel_volume)
 
+        # plot cluster map
```

Comment:
Did you figure out why I have the `try` and `except` clause here?

Reply:
Nope! But added `ValueError` because it's the only reasonable thing `nilearn` would throw here.